CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ if (ENABLE_TESTS)
     enable_testing()
 
     ### Todo: improve this
```

Comment:
```suggestion
```
If still needed, consider creating an issue for it.

Reply:
It is an old issue, I was not the one who created this.